dapp/src/components/LedgerDerivationContent.js

```diff
@@ -78,31 +206,104 @@ const LedgerDerivationContent = ({}) => {
             'Select a Ledger derivation path'
           )}
         </h2>
-        {options.map((option) => {
-          return (
-            <button
-              key={option.path}
-              className="text-center"
-              onClick={() => onSelectDerivationPath(option.path)}
-            >
-              {option.display}
-            </button>
-          )
-        })}
+        <div className={`paths d-flex flex-row`}>
+          {options.map((option) => {
+            return (
+              <button
+                key={option.path}
+                className={
+                  'text-center ' + (activePath === option.path && 'active')
```

Comment:
You can actually utilise `classnames` (its meant for this exactly! and is already used throughout the codebase).
Docs - https://github.com/JedWatson/classnames